### ISSUE TITLE
Add more fixers

### ISSUE THREAD

**Issue by umpirsky** [umpirsky/Symfony-Upgrade-Fixer#4]
This is our TODO list https://github.com/symfony/symfony/blob/2.8/UPGRADE-3.0.md.

**Comment by jean-pasqualini** [comment]
Il y a également ceux-la pour passer de la 2.3 à la 2.7 

https://gist.github.com/mickaelandrieu/5211d0047e7a6fbff925

**Comment by umpirsky** [comment]
@jean-pasqualini Yes. The focus and priority curently is on 3.0. Maybe after we fix https://github.com/umpirsky/Symfony-Upgrade-Fixer/issues/3 it will make sense to also add this older version fixers. Thanks.

**Comment by mickaelandrieu** [comment]
A lot of upgrading guides from my guide are still useful because a lot of projects still use the old (and still good)  2.3 LTS. 

To be honest only open source projects have migrated to the last LTS,  I have never seen a real enterprise project migrated,  mainly because Symfony 2.7 and 2.8 are very close and people probably want their projects to be Symfony3-ready: more easy to sell this migration to customers :)

**Comment by umpirsky** [comment]
Yes, the, game is on.  :)

**Comment by jean-pasqualini** [comment]
Après comme de plus en plus de cms utilise comme base le framework full stack Symfony 2.

Ceux par example qui passe de ezpublish 5.3 à ezpublish 5.4, se retrouve à devoir migrer du Symfony 2.3 en Symfony 2.7.

C'est un des cas où de réel entreprises migrent, c'est d’ailleurs le cas de l'agence de communication ou je suis actuellement en presta.
